src/views/Auction/components/HeaderControl/index.tsx

```diff
@@ -20,12 +21,68 @@ const ControlTitle = styled.div({
   color: '#000629',
 })
 
-const ControlButton = styled.div({
-  fontStyle: 'normal',
+const ControlButton = styled.div<SpaceProps>(
+  () => ({
+    fontStyle: 'normal',
+    fontWeight: 500,
+    fontSize: '14px',
+    lineHeight: '17px',
+    color: '#7B7F93',
+  }),
+  space
```

Comment:
Don't we have a central place for the colors in the css? We should change this after the MVP. Because then we can skin this.

Reply:
@martinkrung, we have a comprehensive styled-components theme in `styles/theme.ts`.